### ISSUE TITLE
[Bug]: Code highlighting working abnormally in 4.5.0

### ISSUE THREAD

**Issue by RisaKirisu** [coder/code-server#5343]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS/Web Information

- Web Browser: Chrome, Samsung Internet
- Local OS: Arch Linux, Android
- Remote OS: Arch Linux
- Remote Architecture: amd64
- `code-server --version`: 4.5.0

### Steps to Reproduce

1. open code-server
2. open a code file

### Expected

Code highlighting should work.

### Actual

It's not working as expected

### Screenshot/Video

In VS Code: 
![image](https://user-images.githubusercontent.com/28692567/179169662-93e8a51d-9f1d-4179-9eed-c54df9468690.png)

In code-server:
![image](https://user-images.githubusercontent.com/28692567/179169737-3463de50-9972-44d4-ad56-931117e5cc03.png)

### Does this issue happen in VS Code or GitHub Codespaces?

- [X] I cannot reproduce this in VS Code.
- [X] I cannot reproduce this in GitHub Codespaces.

### Are you accessing code-server over HTTPS?

- [X] I am using HTTPS.

**Comment by consideRatio** [comment]
@RisaKirisu when you visit your code server, are you using a path like `/code` or such in the url, or is it straight up `/`? There is a fix (https://github.com/coder/code-server/pull/5332) related to a failure affecting users that visits code-server via a path like `/code`.

I was curious if both you and me were users accessing code-server via a `/prefix` in the url like `http://localhost/code` or `https://example.com/code` compared to `http://localhost` or `https://example.com/`.

**Comment by jsjoeio** [comment]
This should be fixed in 4.5.1

<img width="1712" alt="image" src="https://user-images.githubusercontent.com/3806031/179628978-73422cb9-1d77-432f-895e-33406fa77cd8.png">

**Comment by RisaKirisu** [comment]
> @RisaKirisu when you visit your code server, are you using a path like `/code` or such in the url, or is it straight up `/`? There is a fix (#5332) related to a failure affecting users that visits code-server via a path like `/code`.
> 
> I was curious if both you and me were users accessing code-server via a `/prefix` in the url like `http://localhost/code` or `https://example.com/code` compared to `http://localhost` or `https://example.com/`.

Yes. In my setup I access my code-server by 'https://mydomain.xxx/vsc/'. I don't know if this is the cause tho.

**Comment by jsjoeio** [comment]
For the curious, this issue should be fixed via https://github.com/coder/code-server/pull/5332 when a new release comes out.
